scripts/lib/CIME/XML/component.py

```diff
@@ -116,14 +118,74 @@ def _get_value_match(self, node, attributes=None, exact_match=False):
         return match_value
 
     #pylint: disable=arguments-differ
-    def get_description(self, compsetname):
+    def get_description(self, compsetname, comp_class=None):
```

Comment:
Since it doesn't do this for older versions I am leaving the name alone and adding a comment.